meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -84,14 +86,82 @@ function filterLayerMappings(
 	return result
 }
 
+function getFilteredTableValuesFromComparison<DBInterface extends { _id: ProtectedString<any> }>(
+	targetFullAttribute: string,
+	item: ConfigManifestEntrySelectFromTableEntryWithComparisonMappings<boolean>,
+	configPath: string,
+	dbObject: DBInterface,
+	alternateDbObject?: DBInterface
+): SelectOption[] {
```

Comment:
Just a note: This function states in the name that we get the table values, but return select options. I don't know if that inconsistency is an issue.

Reply:
I changed the naming to more correctly display that we are returning `SelectOption`'s.